docs/user_guide/mitigation.rst

```diff
@@ -194,6 +207,684 @@ Postprocessing
+    >>> # Done to prevent overfitting towards the overrepresented class
+    
+    >>> def resample_dataset(X_train, Y_train, sf_train):
+    ...   negative_ids = Y_train[Y_train == 0].index
+    ...   positive_ids = Y_train[Y_train == 1].index
+    ...   np.random.seed(1)
+    ...   balanced_ids = positive_ids.union(np.random.choice(a=negative_ids, size=len(positive_ids)))
+    ...
+    ...   X_train = X_train.loc[balanced_ids, :]
+    ...   Y_train = Y_train.loc[balanced_ids]
+    ...   sf_train = sf_train.loc[balanced_ids]
+    ...
+    ...   return X_train, Y_train, sf_train
+
+    >>> X_train, Y_train, sf_train = resample_dataset(X_train, Y_train, sf_train)
+   
+
+.. doctest:: mitigation
+    :options:  +NORMALIZE_WHITESPACE
+
+    >>> #Train classifier, make and inspect default predictions
+
+    >>> from sklearn.ensemble import RandomForestClassifier
+    >>> from fairlearn.metrics import MetricFrame, false_negative_rate
+    >>> from fairlearn.postprocessing import plot_positive_predictions, plot_proba_distribution
+
+    >>> classifier = RandomForestClassifier(random_state=1).fit(X_train,Y_train) 
+    >>> Y_pred_proba_clf = classifier.predict_proba(X_test)[:, 1]
+    >>> Y_pred_clf = classifier.predict(X_test)
+
+    >>> MetricFrame(metrics=false_negative_rate, 
+    ...             y_true=Y_test,
+    ...             y_pred=Y_pred_clf, 
+    ...             sensitive_features=sf_test).by_group
+    race
+    AfricanAmerican    0.383205
+    Asian              0.414634
+    Caucasian          0.401478
+    Hispanic           0.391304
+    Other              0.359375
+    Name: false_negative_rate, dtype: float64
+
+    >>> plot_positive_predictions(sf_test,Y_pred_clf)    
+
+.. figure:: ../auto_examples/images/user_guide_thresholder_no_thresholds.png
+    :align: center
+
+It can be useful to also plot the distribution of the probabilities obtained
+by calling :code:`predict_proba()[:,1]`. This can give extra insights it what
+effect choosing a different threshold will have on the amount of positive
+predictions. 
+
+.. doctest:: mitigation
+    :options: +NORMALIZE_WHITESPACE
+
+    >>> plot_proba_distribution(sf_test,Y_pred_proba_clf)
+
+.. figure:: ../auto_examples/images/user_guide_thresholder_proba_distr.png
+    :align: center
+
+.. _pos_thresholds:
+
+Two ways of specifying positive thresholds
+*******************************************
+Say, you want to change the threshold of a specific group to 0.4 (all 
+probabilities greater than 0.4 will be predicted as 1). By default,
+:class:`Thresholder` assumes a provided threshold to be positive, which
+means that there are two ways to specify this threshold in the
+:code:`threshold_dict`: 
+
+    - 0.4
+    - ('>', 0.4)
+
+Changing this threshold for groups 'Asian' and 'Other' shows the impact a
+change in threshold of just 0.1 can have on the false negative rates and the
+amount of positive predictions. Note that the purpose of this user guide is
```

Comment:
If I read this as a user I'd be super curious to know what the debate is all about 😆 Why would I not want to do this? I'm not sure if it's easy to capture that in a somewhat concise manner, though...